index.bs

```diff
@@ -9824,6 +9824,63 @@ Methods</h5>
 		</div>
 </dl>
 
+<h5 id="AudioWorkletProcessor-instantiation">
+The instantiation of AudioWorkletProcessor</h5>
+
+The construction of an {{AudioWorkletNode}} is followed by the construction
+of a corresponding {{AudioWorkletProcessor}} instance in the
+{{AudioWorkletGlobalScope}}.
+
+At the end of the {{AudioWorkletNode}} construction,
+<dfn>processor construction data</dfn> will be prepared for the cross-thread
+transfer. This data contains:
+
+<dl>
+	: <dfn dfn>nodeName</dfn>
+	::
+		A {{DOMString}} of the {{AudioWorkletProcessor}} name that is to
+		be looked up <a>node name to processor constructor map</a>.
+	: <dfn dfn>nodePort</dfn>
+	::
+		A serialized {{MessagePort}} that is paired with
+		the associated {{AudioWorkletNode}}'s {{AudioWorkletNode/port}}.
+	: <dfn dfn>nodeOptions</dfn>
+	::
+		A serialized {{AudioWorkletNodeOptions}} dictionary that is
+		given to the {{AudioWorkletNode}}
+		{{AudioWorkletNode()|constructor}}.
+</dl>
+
+Upon the arrival of the transferred data on the {{AudioWorkletGlobalScope}},
+the <a>rendering thread</a> will trigger the algorithm below:
+
+	<div id="invoking-processor-constructor" algorithm="invoking processor constructor">
+		Upon the arrival of <a>processor construction data</a>
+		<var>constructionData</var>, perform the following steps. If
+		any step throws an exception, abort the remaining steps and
+		<a>queue a task</a> to fire an
+		<a href="https://www.w3.org/TR/html50/webappapis.html#the-errorevent-interface">
+		ErrorEvent</a> named <code>processorerror</code> at the
+		{{AudioWorkletNode}} on the <a>control thread</a> with the
+		relevant information about the error.
+
+		1. Let <var>name</var> be the [=nodeName=] member of
+			<var>constructionData</var>.
+
+		1. Let <var>options</var> be the [=nodeOptions=] member of
+			<var>constructionData</var>.
+
+		1. Let <var>port</var> be the [=nodePort=] member of
+			<var>constructionData</var>.
+
+		1. Let <var>processorCtor</var> be the result of looking
+			up <var>name</var> on the {{AudioWorkletGlobalScope}}'s
+			<a>node name to processor constructor map</a>.
+
+		1. Perform Construct(<var>processorCtor</var>,
+			« <var>options</var> ») with the given <var>port</var>.
```

Comment:
What does "with the given port" mean, exactly?

Reply:
This is something I wanted to ask, but you already pointed out the infra spec uses the similar pattern of using `construction data`. I'll take a look at it.

Reply:
No, the infra spec has nothing to say about this....

Reply:
ACK